### DISCUSSION TITLE
[Ubuntu 22.04] Error org.freedesktop.DBus.Error.ServiceUnknown: The name org.asamk.Signal was not provided by any .service files

### DISCUSSION THREAD

**Discussion by cloudengineer89** [AsamK/signal-cli#1379] [Q&A]
I'm trying to send a message via dbus service and I am faced with an error:` Error org.freedesktop.DBus.Error.ServiceUnknown: The name org.asamk.Signal was not provided by any .service files` after the command:
`dbus-send --system --type=method_call --print-reply --dest="org.asamk.Signal" /org/asamk/Signal/_380931112233 org.asamk.Signal.sendMessage string:"Message text goes here" array:string: string:+380662223344`
Please help me to find out where is the problem.

btw when I put `--session` instead of `--system `I've got different error: `Failed to open connection to "session" message bus: /usr/bin/dbus-launch terminated abnormally without any error message`

Here is my result after executing `systemctl status dbus`:
```
root@ip-adress://# sudo systemctl status dbus
● dbus.service - D-Bus System Message Bus
     Loaded: loaded (/lib/systemd/system/dbus.service; static)
     Active: active (running) since Sun 2023-11-12 18:37:48 UTC; 27min ago
TriggeredBy: ● dbus.socket
       Docs: man:dbus-daemon(1)
   Main PID: 21876 (dbus-daemon)
      Tasks: 1 (limit: 1091)
     Memory: 616.0K
        CPU: 9ms
     CGroup: /system.slice/dbus.service
             └─21876 @dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only

Nov 12 18:37:48 ip-adress systemd[1]: Started D-Bus System Message Bus.
Nov 12 18:37:48 ip-adress dbus-daemon[21876]: [system] AppArmor D-Bus mediation is enabled
```

**Comment by nvvetal** [comment]
Ubuntu 23 same

`sudo systemctl status dbus
● dbus.service - D-Bus System Message Bus
     Loaded: loaded (/lib/systemd/system/dbus.service; static)
     Active: active (running) since Tue 2023-11-28 15:50:12 UTC; 1min 10s ago
TriggeredBy: ● dbus.socket
       Docs: man:dbus-daemon(1)
   Main PID: 1465684 (dbus-daemon)
      Tasks: 1 (limit: 4512)
     Memory: 672.0K
        CPU: 15ms
     CGroup: /system.slice/dbus.service
             └─1465684 @dbus-daemon --system --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only

Nov 28 15:50:12 vps-256af249 systemd[1]: Starting dbus.service - D-Bus System Message Bus...
Nov 28 15:50:12 vps-256af249 dbus-daemon[1465684]: [system] AppArmor D-Bus mediation is enabled
Nov 28 15:50:12 vps-256af249 systemd[1]: Started dbus.service - D-Bus System Message Bus.`
